client/src/Week.jsx

```diff
@@ -117,7 +116,7 @@ class Week extends React.Component {
             chosenStyle = normalDateStyle;
           }
 
-          return <td class = 'day'
+          return <td className = 'day' key={index}
```

Comment:
In general, using indexes as a key is an anti-pattern. Probably won't affect your app here, but good to be aware. Linked from React Docs: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Oh wow good to know, thanks!